docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.7'
+services:
+  replication-solr:
+    image: cnxta/replication-solr:0.3.0-SNAPSHOT
+    ports:
+     - "8983:8983"
+    volumes:
+      - replication-data:/var/solr
+    networks:
+      - replication
+    entrypoint:
+      - bash
+      - "-c"
+      - "precreate-core replication_item; precreate-core replication_site; precreate-core replication_status; precreate-core replication_config; exec solr -f"
+  ion-replication:
+    image: cnxta/replication:0.3.0-SNAPSHOT
+    networks:
+      - replication
+    entrypoint:
```

Comment:
:exclamation: no need to override the entrypoint. Its already defined in the `Dockerfile`

Reply:
This is slightly different than the one in the Dockerfile. It specifies the ssl.system.properties. After looking at your other comments though we might be able to make them the same.